app/forms/declaration/GoodsLocation.scala

```diff
@@ -19,55 +19,53 @@ package forms.declaration
 import forms.DeclarationPage
 import play.api.data.Forms.{optional, text}
 import play.api.data.{Form, Forms}
-import play.api.libs.json.Json
+import play.api.libs.json.{Json, OFormat}
 import services.Countries.allCountries
 import utils.validators.forms.FieldValidator._
 
-case class GoodsLocation(
-  typeOfLocation: String,
-  qualifierOfIdentification: String,
-  identificationOfLocation: Option[String],
-  additionalIdentifier: Option[String],
-  addressLine: Option[String],
-  city: Option[String],
-  postCode: Option[String],
-  country: String
-)
+case class GoodsLocation(code: String)
 
 object GoodsLocation extends DeclarationPage {
-  implicit val format = Json.format[GoodsLocation]
 
-  val formId = "GoodsLocation"
+  implicit val format: OFormat[GoodsLocation] = Json.format[GoodsLocation]
```

Comment:
Is this all taken from the movements service?

Reply:
Yes